modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -40,46 +40,49 @@ class BrushingAndLinkingInport;
 class BrushingAndLinkingProcessor;
 /**
  * \class BrushingAndLinkingManager
- * \brief VERY_BRIEFLY_DESCRIBE_THE_CLASS
- * DESCRIBE_THE_CLASS
+ * \brief Manages row filtering, row selection and column selection from multiple sources.
  */
 class IVW_MODULE_BRUSHINGANDLINKING_API BrushingAndLinkingManager {
 public:
     BrushingAndLinkingManager(Processor* p,
                               InvalidationLevel validationLevel = InvalidationLevel::InvalidOutput);
     virtual ~BrushingAndLinkingManager();
-
+    /* 
+     * Return number of selected rows.
```

Comment:
This has nothing to do with rows, this is not only used for data frames. Brushing and linking should know nothing of columns and rows

Reply:
In some sense yes, but we now have the possibility to select columns as well. I can revert these semantic changes

Reply:
How about:
Return number of selected items/rows

Reply:
fixed

Reply:
Can keep the columns for now, but we will have to generalize that at some point are remove the hardkoded names...

Reply:
Ok, then I'll merge these changes. Scatterplots need to take selection into account as well, future PR